bookingCmodule.js

```diff
@@ -0,0 +1,57 @@
+/////Create modalSection3\\\\\
+
+//POST request
+export default async function requestPOST(challengeDataId, inputDate, inputName, inputEmail, selectSlot, selectPlayers) {
+
+    let i = selectSlot.value;
+    let slot = selectSlot[i].innerText;
```

Comment:
This is unnecessarily complex. Instead of having to do this, you could have just saved the slot text as the value of the `option` (see previous comment` which would have allowed you to do this:

```js
const slot = selectSlot.value;
```